code/modules/core_implant/cruciform/machinery/obelisk.dm

```diff
@@ -140,11 +140,12 @@ GLOBAL_LIST_EMPTY(all_obelisk)
 			eotp.scanned |= mob
 			if(I && I.active && I.wearer)
 				eotp.addObservation(20)
-			else if(mob.mutation_index > 2 && !get_active_mutation(mob, MUTATION_ATHEIST) && !get_active_mutation(mob, MUTATION_GODBLOOD))
-				mob.adjustFireLoss(mob.mutation_index)
+			else if(mob.mutation_index > 4 && !get_active_mutation(mob, MUTATION_ATHEIST) && !get_active_mutation(mob, MUTATION_GODBLOOD))
+				var/mutation_damage = mob.mutation_index / 2
+				mob.adjustFireLoss(mutation_damage)
 				eotp.removeObservation(15)
-				if(mob.mutation_index > 5)
-					mob.adjustFireLoss(mob.mutation_index * 2)
+				if(mob.mutation_index > 9)
```

Comment:
I'm the one that suggested this change but that seems quite high. How often do people go past having 4/9 genemods and aberrant organs?

Reply:
Depends on the round. A common upgrade I've seen is adding the durability upgrade to bones, which makes 7 modded organs.